### PR TITLE
Improve Science blog entry 1

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR applies some little Improvements to the first science blog entry.

<details>
<summary>Preview of the changes</summary>

1. Highlighted the "Zwecke"
Bildschirmfoto 2021-06-17 um 21 52 17

2. Added English sentence which points to German version 
Bildschirmfoto 2021-06-17 um 21 52 42
</details>

### CODE REVIEW

File: science/2021-06-15-science-blog-1/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ author: CWA Team
 layout: science
 ---
 
-An English version of this science blog article will follow in due course.
+An English version of this science blog article will follow in due course. Please take a look at the [German version](https://www.coronawarn.app/de/science/2021-06-15-science-blog-1/) until the English version is available.
```
Comment (MikeMcC399) [inline science/2021-06-15-science-blog-1/index.md:10]: It would be better to use a relative URL here since the target is on the same webserver.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Suggest to use relative URL.

File: science/2021-06-15-science-blog-1/index.md (hunk 1 above)
Comment (MikeMcC399) [reply science/2021-06-15-science-blog-1/index.md:10]: I suggest changing
"[German version](https://www.coronawarn.app/de/science/2021-06-15-science-blog-1/)" to
"[German version](../../de/science/2021-06-15-science-blog-1)"

Thanks in any case for the cross-linking! 👍🏻

Comment (Ein-Tim) [reply science/2021-06-15-science-blog-1/index.md]: Done in https://github.com/corona-warn-app/cwa-website/pull/1397/commits/1529bafea7eea61c63cb8d78183a7cefbd7c108c.

Thanks for the reminder!

**Review by dsarkar — APPROVED** [review]

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399  Thanks!

**Pull request merged**